DataAnalytics.md

```diff
@@ -0,0 +1,11 @@
+# Spark on Graviton
+Below are some general guidelines that can be referenced by users trying to improve over all Application performance across Spark clusters. Note that, there is no magic/standard values that works in all use cases and each case can be different. Most of the times, it depends on dataset sizes, cores available, workload type etc. 
```

Comment:
Can you add perf data and also the version of Spark used in your tests/ runs?

Reply:
I added extra details to show the versions used. I avoided adding too many details w.r.t to config options though